recipes/ue

```diff
@@ -0,0 +1,3 @@
+(ue :repo    "unrealemacs/ue.el"
+    :fetcher gitlab
+    :files   ("*.el" "snippets"))
```

Comment:
I believe this is equivalent (and gives your recipe room to grow without another PR):
```suggestion
    :files   (:defaults "snippets"))
```